### DISCUSSION TITLE
Copyright for translations

### DISCUSSION THREAD

**Discussion by samanyougarg** [vedicscriptures/bhagavad-gita-api#5] [General]
Hi @PtPrashantTripathi,

Thank you very much for building this awesome API.

I was wondering if you asked the respective organisations to let you distribute their translations and commentaries via the API?

We would like to use the translations for BhagavadGita.io and before we use them, we just wanted to be sure we are not violating any copyrights.

**Comment by PtPrashantTripathi** [comment]
I haven't took any written permission from respective organisations cause these books are totally available in public domain. 

If I will get any notice from them I will update you also.

Thanks and regards
[Pt. Prashant Tripathi](https://ptprashanttripathi.github.io)
